_posts/2025-11-11-announcing-ballerina-2201.11.0-swan-lake-update-13.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog-post
+title: Announcing Ballerina Swan Lake Update 13 (2201.13.0)
+author: Asma Jabir
+published-date: 11 Nov 2025
+status: Published
+socialmediaimage: ballerina-generic-social-media-image-2023.png
+permalink: /posts/2025-11-11-announcing-ballerina-2201.13.0-swan-lake-update-13/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+The Ballerina team is excited to announce the thirteenth update release of Ballerina Swan Lake, Ballerina Swan Lake Update 13 (2201.13.0). This update introduces experimental natural programming capabilities with first-class syntax for working with LLMs, multi-package workspace support for managing complex projects, decoupled CLI tools for faster updates, and intelligent build caching for improved developer productivity.
+
+## Natural expressions
+
+Update 13 introduces natural expressions, an experimental feature that enables first-class syntax for [natural language](https://blog.ballerina.io/posts/2025-04-26-introducing-natural-programming/) integrations. This enables developers to [work with Large Language Models (LLMs)](https://ballerina.io/learn/work-with-llms-using-natural-expressions/) directly within Ballerina code.
+
+Natural expressions are blocks of natural language instructions executed at runtime with the help of generative AI models. They are dependently typed, allowing the model to identify the required result format based on the expected type and automatically bind the response.
+
+This feature supports multimodal inputs, handling document types such as images, audios and files.
+
+<img alt="Multimodal inputs" src="/images/u13/multimodal-inputs-np.png">
+
```

Comment:
```suggestion
For more information on the wider concept proposed as "Natural Programming", see [Natural Language is Code: A hybrid approach with Natural Programming](https://blog.ballerina.io/posts/2025-04-26-introducing-natural-programming/)

```

Reply:
Addressed in [56630af](https://github.com/ballerina-platform/blog/commit/56630afc811e153d8dc1de706b1fd8a4cedc25d0)